tests/storage/test_transactions.py

```diff
@@ -57,8 +56,11 @@ def test_initial_set_transactions(self) -> None:
         self.get_success(d)
 
     def test_large_destination_retry(self) -> None:
+        max_retry_interval = (
+            self.hs.config.federation.destination_max_retry_interval_ms * 1000
```

Comment:
I'm confused why we multiply by `1000` here. `set_destination_retry_timings(...)` expects time in `ms` but we seem to be converting it to seconds.

https://github.com/matrix-org/synapse/blob/224ef0b669fdd85925d66deb38ba1b51c5aaa1bd/synapse/storage/databases/main/transactions.py#L220-L222